ansible/templates/sir_complainsalot-params-prod.json.j2

```diff
@@ -46,6 +46,18 @@
       "max_age":6,
       "maintainers": ["U01AP8GSWG3","U01069KCRS7"],
       "retired-signals": ["raw_pct_negative","smoothed_pct_negative","raw_tests_per_device","smoothed_tests_per_device"]
+    },
+    "nchs-mortality": {
+      "max_age":13,
```

Comment:
Just checking if max_age is similar to the max_lag_in_days argument in the indicator lags - if yes, I believe that NCHS has lags of up to 15/16?

Reply:
ooh interesting. Yes it should generally be the same (or one day greater to allow for acquisition lag), but I was going off the most recent lag pattern from the last month, which has been 12 at most over the ~3 weekly cycles that the indicator has been logging max lags.

![image](https://user-images.githubusercontent.com/1158666/126338700-057e176c-2472-4e93-9576-13a402ce98bb.png)

Is the 15-16 figure coming from some other output I should be looking at, or from the indicator logic?

Reply:
Hm, I downloaded the info from most likely the same source and got up to 16:

![image](https://user-images.githubusercontent.com/37101453/126340032-5cc65ea2-faed-49c6-8849-03ca2362275f.png)

The graph values actually seem to resemble what I have, but there's a truncating at 12. Not too sure why...

Reply:
The search I used was something like log.file.path:/var/log.indicators/*.log AND message: max_lag_in_days. It's also 7 days old, so many mine's just outdated.

Reply:
serves me right, turns out I set a max of 12 in the graph y-axis limits, and kibana doesn't make it obvious when you've got data that exceeds the limits 😖 

17 it is!